<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateNewCustomer</class>
 <widget class="QDialog" name="CreateNewCustomer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="endCity_select">
    <item>
     <widget class="QLabel" name="endCity_label">
      <property name="text">
       <string>终点城市：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="end_City">
      <item>
       <property name="text">
        <string>北京</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>天津</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>哈尔滨</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>大连</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>乌鲁木齐</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>成都</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>六安</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>银川</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>呼和浩特</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>威海</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>济南</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>西安</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="Strategy_frame">
    <item>
     <widget class="QRadioButton" name="leastTime_Strategy">
      <property name="text">
       <string>最少时间策略</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="leastFee_Strategy">
      <property name="text">
       <string>最少费用策略</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="leastTime_Fee_Strategy">
      <property name="text">
       <string>限时最少费用策略</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="limited_Time">
      <property name="maximum">
       <number>240</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="prompt_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>尊敬的旅客您好！请在下面填写您的旅游信息：</string>
   </property>
  </widget>
  <widget class="QLabel" name="strategy_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>策略选择：</string>
   </property>
  </widget>
  <widget class="QLabel" name="passCities_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>途径城市以及预计的停留时间(单位/小时)：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createCustomer_Button">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="logoTravel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>0</y>
     <width>161</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>142</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="startCity_select">
    <item>
     <widget class="QLabel" name="startCity_label">
      <property name="text">
       <string>出发城市：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="start_City">
      <property name="toolTip">
       <string/>
      </property>
      <item>
       <property name="text">
        <string extracomment="Beijing">北京</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>天津</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>哈尔滨</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>大连</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>乌鲁木齐</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>银川</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>呼和浩特</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>威海</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>成都</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>济南</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>西安</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>六安</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>台北</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>4</number>
   </property>
   <property name="text">
    <string>旅客注册</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>211</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QCheckBox" name="Taibei_Button">
      <property name="text">
       <string>台北</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="Taibei_Time">
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="maximum">
       <number>240</number>
      </property>
      <property name="singleStep">
       <number>2</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>488</width>
     <height>78</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QCheckBox" name="Beijing_Button">
          <property name="text">
           <string>北京</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="Chendu_Button">
          <property name="text">
           <string>成都</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="Dalian_Button">
          <property name="text">
           <string>大连</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QSpinBox" name="Beijing_Time">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>240</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="Chendu_Time">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>240</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="Dalian_Time">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>240</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QCheckBox" name="Tianjin__Button">
          <property name="text">
           <string>天津</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="Luan_Button">
          <property name="text">
           <string>六安</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="Yinchuan_Button">
          <property name="text">
           <string>银川</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QSpinBox" name="Tianjin_Time">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>240</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="Luan_Time">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>240</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="Yinchuan_Time">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>240</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="HaerBin_Button">
          <property name="text">
           <string>哈尔滨</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="Jinan_Button">
          <property name="text">
           <string>济南</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="HuheHaoTe_Button">
          <property name="text">
           <string>呼和浩特</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QSpinBox" name="HaerBin_Time">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>240</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="Jinan_Time">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>240</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="HuheHaoTe_Time">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>240</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="WuluMuQi_Button">
          <property name="text">
           <string>乌鲁木齐</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="Xian_Button">
          <property name="text">
           <string>西安</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="WeiHai_Button">
          <property name="text">
           <string>威海</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QSpinBox" name="WuluMuQi_Time">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>240</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="Xian_Time">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>240</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="WeiHai_Time">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>240</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
